cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -563,20 +571,28 @@ private String getSecurityProtocol(boolean tls, boolean sasl)   {
         return options;
     }
 
-    static void addOption(List<String> options, String option, String value) {
-        if (value != null) options.add(String.format("%s=\"%s\"", option, value));
+    static void addOption(Map<String, String> options, String option, String value) {
```

Comment:
I know you did not changed this. But should it be renamed to `addOptionIfNotNull`?

Reply:
I'm wondering why would we do that? as this is not a test method and as not all the methods we have that contain null checker are called that way ? so why would we call this method that way ? if you can clear that to me ?

Reply:
Well, it has the null check. So it adds the option only if it is not null. That is important information as it for example means you do not need to have a null check on the outside of calling this method. So it would be good ot have it clear from the name.

Reply:
I see, thanks for the clarification